File: src/main/java/itmo/is/project/mapper/module/dock/DockModuleMapper.java
package itmo.is.project.mapper.module.dock;

import itmo.is.project.dto.module.dock.DockModuleDto;
import itmo.is.project.mapper.EntityMapper;
import itmo.is.project.model.module.dock.DockModule;
import org.mapstruct.Mapper;

@Mapper(componentModel = "spring", uses = {DockModuleBlueprintMapper.class})
public interface DockModuleMapper extends EntityMapper<DockModuleDto, DockModule> {
}

File: src/main/kotlin/itmo/isproject/mapper/module/dock/DockModuleMapper.kt
package itmo.isproject.mapper.module.dock

import itmo.isproject.dto.module.dock.DockModuleDto
import itmo.isproject.mapper.EntityMapper
import itmo.isproject.model.module.dock.DockModule
import org.mapstruct.Mapper

@Mapper(uses = [DockModuleBlueprintMapper::class])
interface DockModuleMapper : EntityMapper<DockModuleDto, DockModule>
